tests/acceptance/features/apiSpacesShares/publicLinkDownload.feature

```diff
@@ -0,0 +1,43 @@
+@api @skipOnOcV10
+Feature: Public can download folders from project space public link
+  As a public
+  I want to be able to download folder from public link
+  So that I can gain access to it's contents
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+    And using spaces DAV path
+    And the administrator has given "Alice" the role "Space Admin" using the settings api
+    And user "Alice" has created a space "new-space" with the default quota using the GraphApi
+
+
+  Scenario: download a folder from public link of a space
+    Given user "Alice" has created a public link share of the space "new-space" with settings:
+      | permissions | 1                        |
+      | name        | someName                 |
+    And user "Alice" has created a folder "NewFolder" in space "new-space"
+    And user "Alice" has uploaded a file inside space "new-space" with content "some content" to "NewFolder/test.txt"
+    When public downloads the folder "NewFolder" of space "new-space" from the last created public link of "Alice" using the resource id
```

Comment:
Need suggestions on if this is an appropriate way to do this. Realistically, the public would never know the `resource-id` of the file
 But the endpoint used to download the folders uses the `archiver` endpoint with query params `public-token` and `fileid`

Reply:
Or maybe the step can be only `When public downloads the folder "NewFolder" of space "new-space" from the last created public link of "Alice"`
And in step def, we can get the resource id. Also, I think the UI part may also does the same internally (but not sure - should be the same)

Reply:
@phil-davis @individual-it

Reply:
In a real situation, "the public" has the link token. "The public" does a PROPFIND using the token (or some similar "get a list of resources" request), and a list of the resources (file and folders) in the public link share is returned. The data returned in the PROPFIND should have resource names and ids. If it does not have the id, then "the public" has to do another PROPFIND to the resource that they want, and get the id from that.

Does something like this work for "the public"?

If it does, then there is a way for "the public" to get the resource id. The test step can do the same sequence of requests. That will demonstrate that "the public" can really find out the file id, and then use it to access the `archiver` endpoint.

Reply:
@phil-davis Thanks I'll try that and see if it works

Reply:
@phil-davis turns out we can propfind as a public with public files api https://doc.owncloud.com/server/next/developer_manual/webdav_api/public_files.html 
I've updated the tests as well